.travis.yml

```diff
@@ -29,6 +29,9 @@ matrix:
 # we'll put the Kodi source on the same level
 #
 before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; apt-get update; apt-get install libtinyxml2-dev; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install tinyxml2; fi
```

Comment:
This is wrong, will give you the correct changes later.

Reply:
Well, I'm curious how it should be :)

Reply:
see https://github.com/rbuehlma/pvr.zattoo/compare/cleanup...Rechi:cleanup again
I overlooked that `CMAKE_DEBUG_POSTFIX` is set for all platforms. Usually libs only set that for windows.